docs/unify_command.md

```diff
@@ -0,0 +1,13 @@
+### Unify
```

Comment:
Add an explanation for why and when they would like to use it.

Reply:
done